src/aktualizr_repo/image_repo.cc

```diff
@@ -37,11 +37,14 @@ void ImageRepo::addBinaryImage(const boost::filesystem::path &image_path, const
   target["hashes"]["sha256"] = boost::algorithm::to_lower_copy(boost::algorithm::hex(Crypto::sha256digest(image)));
```

Comment:
I know that it's not related to this specific PR, but loading the whole file/image into memory doesn't look like optimal way to calculate file hash, unless aktualizr-repo is used just for testing purposes.

Reply:
Valid point. This is not what we do in libaktualizr itself, but aktualizr-repo is really just for testing, so it's not so critical.